independent-projects/tools/registry-client/src/main/java/io/quarkus/registry/catalog/json/JsonPlatform.java

```diff
@@ -35,15 +38,30 @@ public void setName(String name) {
     @Override
     @JsonDeserialize(contentAs = JsonPlatformStream.class)
     public List<PlatformStream> getStreams() {
-        return streams == null ? Collections.emptyList() : streams;
+        return streams == null ? Collections.emptyList() : new ArrayList<>(streams.values());
```

Comment:
We probably don't need to return a List here. Then we wouldn't need `new ArrayList()`. I guess I used `list.get(0)` for `getRecommendedXXX()` but we could add a `recommendedStream` field which could be initialized to the first stream added to the platform and a `getRecommendedStream()` method that would return its value?
We still need to preserve the order in which the streams are added.

Reply:
This override the interface method, and I did not want to change the current interface methods. There is already a `getRecomendedStream` method. The order will be preserved.